modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -40,3 +41,89 @@ Select one of the configuration options below:
 <1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany PersistentVolumeClaim (PVC). This is necessary if you do not have dynamic volume provisioning or want to have more control over how the PersistentVolume is created.
 
 <2> Set `storage.hive.sharedPVC.createPVC` to `true` and set the `storage.hive.sharedPVC.storageClass` to the name of a StorageClass with ReadWriteMany access mode. This will use dynamic volume provisioning to have a volume created automatically.
+--
+
+.  Create the following resource objects that are required to deploy an NFS server for metering. Use the `oc create -f <file-name>.yaml` command to create the object YAML files.
+..  Configure a PersistentVolume resource object. For example, `nfs_persistentvolume.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nfs
+  labels:
+    role: nfs-server
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+  - ReadWriteMany
+  storageClassName: nfs-server
+  nfs:
+    path: "/"
+    server: REPLACEME
+  persistentVolumeReclaimPolicy: Delete
+----
+..  Configure a Pod resource object with the `nfs-server` role. For example, `nfs_server.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nfs-server
+  labels:
+    role: nfs-server
+spec:
+  containers:
+    - name: nfs-server
+      image: quay.io/openshifttest/nfs-server
```

Comment:
@lbarbeevargas I'm not sure if this image is public or not.

Reply:
Ah, thanks @pruan-rht. Can you recommend a public image or dummy text?

Reply:
@pruan-rht @lbarbeevargas Seems public, I reached it with private browsing window (incognito) and not on VPN.

Reply:
FWIW, even if the image is public, I'm not sure it's the best image to recommend users install.